src/plugins/multiline/index.ts

```diff
@@ -136,6 +144,14 @@ export default class Multiline extends PluginController<Config> {
           skipWidth: minWidth,
           minPriority: 0,
           maxPriority: 1,
+          spacesToNewlines: this.settings.spacesToNewlines,
+          determineLineBreaksAutomatically:
+            this.settings.automaticallyMultilinify &&
+            (this.settings.disableAutomaticLineBreaksForHandAlignedExpressions
+              ? !(
+                  f?.parentElement?._mqMathFieldInstance?.latex?.() as string
+                ).match(/\\ \\ \\/g)
```

Comment:
Instead of `str.match(/\\ \\ \\/g)`, I'd prefer `/\\ \\ \\/`.test(str) since that is intended to return a boolean.

Reply:
Oh good point, I didn't know there was a function that did that lol

Reply:
fixed